src/app/component/MissionComponent/AcceptMissionButton.jsx

```diff
@@ -4,14 +4,16 @@ import CheckIcon from "@material-ui/icons/Check";
 import Mission from "../../model/Mission";
 
 /**
- * @param {userUID} the user UID that accept the mission
+ * @param {handleUpdatedMissions} function that updates missions tied to a user.
  * @param {user} the user that accept the mission
  * @param {mission} the mission to be accepted for
  */
-const AcceptMissionButton = ({ mission, user }) => {
+const AcceptMissionButton = ({ handleUpdatedMissions, mission, user }) => {
```

Comment:
So this button along with `DeliverMissionButton` `StartMissionButton` and `UnassignMeButton` could all be the component.
Since you're not really styling the button in any special way you probably don't need these at all.
and really this also a good example for composition. 

See here
```js

({mission, ...}) => {

const acceptHandler = () => Mission.accept(mission, user, ...);

...

return <MissionCard>
   <Button {...someProps} onClick={acceptHandler}>
     Accept
   </Button>
   <Button {...someOtherProps} onClick={deliverHandler}>
     Deliver
   </Button>
...
</MissionCard>
}
```

Reply:
What were you're thoughts on this?

Reply:
Also looks like we can remove the `handleUpdatedMissions` prop

Reply:
yeah agree